README.md

```diff
@@ -434,7 +459,8 @@ The NativeScript CLI calls the SDK for the selected target platform and uses it
 
 When you build for Android, the NativeScript CLI saves the application package as an `APK` in `platforms` &#8594; `android` &#8594; `bin`.
 
-When you build for iOS, if the `--device` flag is not set, the NativeScript CLI builds your project for the native emulator and saves the application package as an `APP` in `platforms` &#8594; `ios` &#8594; `build` &#8594; `emulator`. If the `--device` flag is set, the NativeScript CLI builds your project for device and saves the application package as an `IPA` in `platforms` &#8594; `ios` &#8594; `build` &#8594; `device`.
+When you build for iOS, the NativeScript CLI will either build for a device, if there's a device attached, or for the native emulator if there are no devices attached. In order to trigger a native emulator build with an attached device, the `--emulator` flag has to be set.
```

Comment:
In order to trigger a native emulator build with an attached device, the `--emulator` flag has to be set. -> To trigger a native emulator build when a device is attached, set the `--emulator` flag.